### PR TITLE
Use enum34 backport when Enum is not available

### PR DESCRIPTION
**Pull request by Nabellaleen**
Tests were broken in Python < 3.4, this PR fix its

### CODE REVIEW

**Comment by Nabellaleen** [comment]
I added a commit fixing build issues, but they are mainly related to a wrong requirements management in setup.py and broken dependancies in graphql-core

All these things should be fixed propelry soon by work from @jnak on tox migration and from a PR on graphene / graphql-core

**Comment by jnak** [comment]
@Nabellaleen I'm almost done with the migration to tox. I'll be sending a PR tomorrow because I'm off today.

**Comment by Nabellaleen** [comment]
@jnak have a look at the change I made in this PR to take it into account in yours (but it's probably already the case ;))

**Comment by jnak** [comment]
@Nabellaleen Sounds good. I'll take a look tomorrow morning

File: setup.py [hunk 1]
```diff
@@ -10,6 +10,17 @@
         ast.literal_eval(_version_re.search(f.read().decode("utf-8")).group(1))
     )
 
+requirements = [
+    "six>=1.10.0",
+    "graphene>=2.1.3",
+    "SQLAlchemy",
+    "singledispatch>=3.4.0.3",
+    "rx<3.0.0",
```
Comment (Nabellaleen) [inline setup.py]:
```suggestion
    "rx<3.0.0",  # Rx 3.0 or above are only compatible with Python >= 3.6
```

**Comment by jnak** [comment]
@Nabellaleen In case you're curious, here are the changes I worked on yesterday. In order to avoid duplicating work, I think it makes sense to hold on this PR for now and wait for my changes to be merged in. I'll be sending a PR tomorrow. Cheers

https://github.com/jnak/graphene-sqlalchemy/commit/a4d5700896360ebf04ba4f3240ec92d0907fcab3

**Comment by Nabellaleen** [comment]
@jnak Yeah, I see ;) Great job ! I can't wait to see things move :D

**Comment by jnak** [comment]
@Nabellaleen Thanks for re-enabling python 2.7. 

Out of curiosity, what's the policy to merge PRs in this project? It looks like you were able to merge this PR without getting it approved by someone else.

**Comment by Nabellaleen** [comment]
@jnak Indeed, rules seems not enforced in the repository settings - I'll fix it !

For the graphene organisation, the rules were defined some weeks ago during a meeting, and are specific for each repository (between 1 and 3 approvals for a merge).

For graphene-sqlalchemy repository, it's 1 approval, so I should have been blocked for merging this one.

**Pull request merged**